src/test/groovy/io/jsonwebtoken/JwtParserTest.groovy

```diff
@@ -502,7 +512,8 @@ class JwtParserTest {
         String compact = Jwts.builder().setSubject(subject).signWith(SignatureAlgorithm.HS256, key).compact()
 
         try {
-            Jwts.parser().setSigningKey(key).parseClaimsJws(compact);
+            Jwts.parser().setSigningKey(key).parsePlaintextJws(compact)
+            fail()
```

Comment:
This is the test that was actually busted, but we didn't know because of the missing `fail()`